extras/prometheus/gmp/rules.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+apiVersion: monitoring.googleapis.com/v1
+kind: Rules
+metadata:
+  name: uptime-rule
+spec:
+  groups:
+  - name: Micro services uptime
+    interval: 60s
+    rules:
+    - alert: BalancereaderUnavaiable
+      expr: probe_success{job="balancereader-probe"} == 0
+      for: 1m
+      annotations:
+        summary: Balance Reader service is unavailable
+        description: Check Balance Reader pods and it's logs
+      labels:
+        severity: 'critical'
+    - alert: ContactsUnavaiable
+      expr: probe_success{job="contacts-probe"} == 0
+      for: 1m
+      annotations:
+        summary: Contacs service is unavailable
+        description: Check Contacs pods and it's logs
+      labels:
+        severity: 'warning'
+    - alert: FrontendUnavaiable
+      expr: probe_success{job="frontend-probe"} == 0
+      for: 1m
+      annotations:
+        summary: Frontend service is unavailable
+        description: Check Frontend pods and it's logs
+      labels:
+        severity: 'critical'
+    - alert: LedgerwriterUnavaiable
+      expr: probe_success{job="ledgerwriter-probe"} == 0
+      for: 1m
+      annotations:
+        summary: Ledger Writer service is unavailable
+        description: Check Ledger Writer pods and it's logs
+      labels:
+        severity: 'critical'
+    - alert: TransactionhistoryUnavaiable
+      expr: probe_success{job="transactionhistory-probe"} == 0
+      for: 1m
+      annotations:
+        summary: Transaction History service is unavailable
+        description: Check Transaction History pods and it's logs
+      labels:
+        severity: 'critical'
+    - alert: UserserviceUnavaiable
+      expr: probe_success{job="userservice-probe"} == 0
+      for: 1m
+      annotations:
+        summary: User Service is unavailable
```

Comment:
`Service` -> `service`. Have it consistently simple case

Reply:
Updated name the same way as it is declared in the `README.md` files for each microservice.